xds/src/main/java/io/grpc/xds/BootstrapperImpl.java

```diff
@@ -215,7 +215,8 @@ BootstrapInfo bootstrap(Map<String, ?> rawData) throws XdsInitializationExceptio
         certProviders.put(name, certificateProviderInfo);
       }
     }
-    String grpcServerResourceId = JsonUtil.getString(rawData, "grpc_server_resource_name_id");
+    String grpcServerResourceId =
```

Comment:
You should also change the property name in `BoootstrapInfo` and everywhere else.

Reply:
The names are really internal names between the Bootstrapper and the consumer (XdsClientWrapperForServerXds) so not strictly an API exposed to outside. But I will change it

Reply:
BTW, what name will work for you? Is `serverListenerResourceNameTemplate` okay?